code/wdb/writedown.q

```diff
@@ -0,0 +1,14 @@
+/-endofperiod function to savedown by integer
+endofperiod:{[currp;nextp;data]
+	.lg.o[`endofperiod;"Received endofperiod. currentperiod, nextperiod and data are ",(string currp),", ", (string nextp),", ", .Q.s1 data]
+        / Need to download sym file to scratch directory if this is Finspace application
+        .lg.o[`createchangeset;"downloading sym file to scratch directory for ",.finspace.database];
+	.aws.get_latest_sym_file[.finspace.database;1_string .wdb.savedir];
+        /- change to integer partition
+	.wdb.currentpartition:(`long$currp);
+	/-call function to savedown tables
+	.wdb.savetables[.wdb.savedir;.wdb.currentpartition;1b;] each .wdb.tablelist[];
+	/- create changeset containing data
+	changeset:.finspace.createchangeset[.finspace.database];
```

Comment:
We don't need to save the changeset id from this function (most likely). Eugene can always add it back in if he finds he needs it, but for your change and as it stands now, it's not needed.